server/models/Requests.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  sitter_id: {
+    type: String,
+    unique: true,
+  },
```

Comment:
I would utilize mongoose `ObjectId` type here as opposed to string, also conventionally JavaScript does not do snake case but instead make use of camelCase.

Example: 
```js
user: {
  type: mongoose.Types.ObjectId,
  required: true,
  ref: 'user'
}
```

Reply:
You can follow the same for sitter as well.

Reply:
This is an excellent recommendation, I have made those changes!